### PR TITLE
update popup modifier description

### PR DESCRIPTION
**Pull request by slavaleleka**

### CODE REVIEW

**Review by sxgunchenko — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
 
 AdGuard will try to close the browser tab with any address that matches a blocking rule with this modifier. Please note that not all the tabs can be closed.
 
+> **Note**
+>
+> It may not work if the popped up page is cached by the browser.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:327]: popped up page?

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
 **Examples**
 
 * `||domain.com^$popup` — if you try to go to `http://domain.com/` from any page in the browser, a new tab in which specified site has to be opened will be closed by this rule.
 
-> It may not work if the popped up page is cached by the browser. It also will not work with some tricky popup methods. In such cases, it is better to use [AdGuard Popup Blocker](https://github.com/AdguardTeam/PopupBlocker) extension.
-
-> **Note**
+> **Compatibility with different versions of AdGuard**
+>
+> `$popup` modifier is reliable in AdGuard Browser extension.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:335]: I guess it's available, not reliable
AdGuard Browser Extension

File: docs/general/ad-filtering/create-own-filters.md [hunk 3]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
+> In AdGuard for Safari and iOS, `$popup` rules will simply block the page right away.
 >
-> Unlike with AdGuard Browser extension, `$popup` modifier is very unreliable when used with AdGuard for Windows, Mac and Android. In AdGuard for Safari and iOS, `$popup` rules will simply block the page right away.
+> When it is used with AdGuard for Windows, Mac and Android, it is very unreliable.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:338]: what do you mean by unreliable?

File: docs/general/ad-filtering/create-own-filters.md [hunk 4]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
+> `$popup` modifier applies `document` content type with special flag which is passed to blocking page.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:339]: `$popup` modifier applies the `document` content type with a special flag to a blocked page.

File: docs/general/ad-filtering/create-own-filters.md [hunk 5]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
+> Blocking page itself can do some checks and close the window if it is really popup.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:340]: if it is really a popup.

File: docs/general/ad-filtering/create-own-filters.md [hunk 6]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
+> Otherwise, page should be loaded.
+> It can be combined with other request type modifiers, as well as `$third-party` and `$important`.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:342]: as well as —> such as

File: docs/general/ad-filtering/create-own-filters.md [hunk 7]
```diff
@@ -322,15 +322,26 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
...
+> However, blocking page may not detect popup in some cases, so it is recommended to use
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:343]: blocked page (блокируемая страница же?)

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 1 above)
Comment (Alex-302) [reply docs/general/ad-filtering/create-own-filters.md:327]: открытая в новом окне/попапе страница. Для тех кто делает фильтры - понятно)

File: docs/general/ad-filtering/create-own-filters.md (hunk 3 above)
Comment (Alex-302) [reply docs/general/ad-filtering/create-own-filters.md:338]: sometimes opened popups cannot be blocked (when content of popup bloked, and popup window closed) by this modifier when used the app

Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:338]: > may not detect a popup in some cases

that's why unreliable

File: docs/general/ad-filtering/create-own-filters.md (hunk 1 above)
Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:327]: может так: `if the popup page` ?

File: docs/general/ad-filtering/create-own-filters.md (hunk 3 above)
Comment (el-termikael) [reply docs/general/ad-filtering/create-own-filters.md:338]: probably, it's better to say this way:

`$popup` modifier works best in AdGuard Browser Extension.
In AdGuard for Safari and iOS, `$popup` rules simply block the page right away.
In AdGuard for Windows, Mac, and Android, `$popup` modifier may not detect a popup in some cases and it won't be blocked.

File: docs/general/ad-filtering/create-own-filters.md [hunk 8]
```diff
@@ -332,13 +332,12 @@ AdGuard will try to close the browser tab with any address that matches a blocki
...
 > **Compatibility with different versions of AdGuard**
 >
-> `$popup` modifier is reliable in AdGuard Browser Extension.
-> In AdGuard for Safari and iOS, `$popup` rules will simply block the page right away.
->
-> When it is used with AdGuard for Windows, Mac and Android, it is very unreliable.
-> `$popup` modifier applies the `document` content type with a special flag which is passed to a blocking page.
-> Blocking page itself can do some checks and close the window if it is really a popup.
-> Otherwise, page should be loaded.
+> 1. `$popup` modifier works best in AdGuard Browser Extension.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:335]: I guess 1. is not supposed to be here and in two strings below)

Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:335]: it looks better 
https://uploads.adguard.org/slbdzdrvtmenn.png

comparing to plain text
https://uploads.adguard.org/slbdzdrvtajrb.png

different behaviour for different list items

Comment (sxgunchenko) [reply docs/general/ad-filtering/create-own-filters.md:335]: but it's `1.` at each point

**Review by sxgunchenko — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 8 above)
Comment (sxgunchenko) [reply docs/general/ad-filtering/create-own-filters.md:335]: but it's `1` at each point

Comment (sxgunchenko) [reply docs/general/ad-filtering/create-own-filters.md:335]: but it's `1` at each point

**Review by sxgunchenko — DISMISSED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 8 above)
Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:335]: it is fine
https://google.github.io/styleguide/docguide/style.html#lists

**Pull request merged**